src/scf-release/src/acceptance-tests-brain/test-scripts/testutils.rb

```diff
@@ -145,13 +145,22 @@ def wait_for_namespace(namespace, sleep_duration=10)
             name, readiness, status, restarts, age = line.split
             next if status == 'Completed'
             next if status == 'Running' && /^(\d+)\/\1$/ =~ readiness
+            puts "# Waiting for: #{line}"
```

Comment:
This probably gives too many lines in normal testing? Can we do tricks with `\r` to not spam the output too much?

Reply:
In normal testing we should not see anything except when we are explicitly verbose.
IMHO this can stay as-is